### ISSUE TITLE
Allow to show only extra markers

### ISSUE THREAD

**Issue by Nadyita** [bfabiszewski/ulogger-server#237]
Hey, just installed your software, and loving the simplicity so far! The one thing I find confusing is that every track point is displayed with a marker. I have a very good external GNSS device that I use for precise measuring of logging trails, and tracks in general, but I have to set the minimum logging interval to 1s, and the minimum logging distance to none in order to get the precision that I need. That looks ridiculous on the map though, especially when I take picture and want to find the markers for the images.

Is it possible to add an option to disable showing non-extra markers on the map? I hacked this into the JS code as a quick fix, but there should be a proper option in the settings to turn this on and off.

**Comment by bfabiszewski** [comment]
Yes, displaying tracks with such big density of track points can be problematic. It is not only visual issue, but also may impact browser performance.
The visual aspect, as you noticed, is not hard to adjust. I plan to do it in the next version. Currently you can only turn off all the markers, not non-extra ones.
The performance problem is more complex. It is on my todo list but requires more time. The solution would be not to show all the points at once. Depending on zoom level the nearby points should be interpolated for display.
